src/lotus/client.rs

```diff
@@ -51,6 +51,12 @@ mod methods {
 
 /// The default gateway actor address
 const GATEWAY_ACTOR_ADDRESS: &str = "f064";
+/// The default state wait confidence value
+const STATE_WAIT_CONFIDENCE: u8 = 5;
+/// We dont set a limit on the look back epoch, i.e. check against latest block
+const STATE_WAIT_LOOK_BACK_NO_LIMIT: i8 = -1;
+/// We are not replacing any previous messages
+const STATE_WAIT_ALLOW_REPLACE: bool = true;
```

Comment:
I am wondering if we should set this to `false`. Having it to `true` means that if the message is replaced by a new one but with different parameters and it is validated, `StateWaitMsg` for this message will also shoe as a success.

It may also be a good idea to update the comment accordingly.

Reply:
when I set it to `false`, it gives this error:https://github.com/consensus-shipyard/lotus/blob/35049ce3171087b9e0d64724332fa20cf3776341/chain/stmgr/searchwait.go#L267. Maybe I have done something wrong?